entry/src/block_component.rs

```diff
@@ -230,28 +230,7 @@ pub enum VersionedBlockMarker {
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum BlockMarkerV1 {
     BlockFooter(VersionedBlockFooter),
-}
-
-/// Version 2 block marker with extended functionality.
-///
-/// Supports all V1 features plus UpdateParent for optimistic block packing in Alpenglow.
-///
-/// # Serialization Format
-/// ```text
-/// ┌─────────────────────────────────────────┐
-/// │ Variant ID                   (1 byte)   │
-/// ├─────────────────────────────────────────┤
-/// │ Byte Length                  (2 bytes)  │
-/// ├─────────────────────────────────────────┤
-/// │ Variant Data              (variable)    │
-/// └─────────────────────────────────────────┘
-/// ```
-///
-/// The byte length field indicates the size of the variant data that follows,
-/// allowing for proper parsing even if unknown variants are encountered.
-#[derive(Debug, Clone, PartialEq, Eq)]
-pub enum BlockMarkerV2 {
-    BlockFooter(VersionedBlockFooter),
+    BlockHeader(VersionedBlockHeader),
```

Comment:
Should we do:
BlockFooter(VersionedBlockFooter) = 0,
BlockHeader(VersionedBlockHeader) = 1,
...
?